### ISSUE TITLE
Not receiving notifications

### ISSUE THREAD

**Issue by Offerel** [LucasGGamerM/moshidon#532]
**Describe the bug**
For some reason, I don't receive notifications with Moshidon. The app is installed from the playstore, notifications are allowed as Android privilege. With the original app, it's working fine.

**To reproduce**
1. Sign in with account in Moshidon
2. Close apo
3. Sign in with another account on other instance
4. Send message to moshidon account
5. No notification appears
6. When opening Moshidon, i can see a counter on icon in the bottom bar
7. Logging out and in Moshidon doesn't change the behavior

**Does this happen in the official app?**
No

**Version**
Moshidon version: [v2.3.0+fork.107.moshinda-play]

**Update:**
I have tested the same behavior with a account on social.tchncs.de and mastodon.social

**Comment by spiezmaestro** [comment]
I experience the same in Moshidon, but also in Megalodon. In Moshidon I connect to a different instance than in Megalodon. Maybe that gives an idea where to look?

**Comment by gaussandhisgun** [comment]
i've had this happen to me in Moshi a few ROMs ago, the in-place duct-taped solution i came up with was installing a [Google Play based UnifiedPush provider](https://apt.izzysoft.de/fdroid/index/apk/org.unifiedpush.distributor.fcm) and switching Moshi to UnifiedPush instead, but after doing that most notifications started coming up twice or thrice. well, they *did* come up, that's what matters - so the issue is somewhere in the playstore notification handler side

**Comment by danvolchek** [comment]
I also noticed this recently. I use the Google play version + mastodon.social.

**Comment by jotaluissoy** [comment]
It seems to be related to MastoHost and Google Play Services.

**Comment by jotaluissoy** [comment]
It's also good to add that it is also happening to the Official app.

**Comment by SquareDot3301** [comment]
Exactly the same problems with Moshidon and the official app.... very weird... I try to uninstall / reinstall / logout / login / restart my phone but it doesn't work... I try with Ntfy for the push notifications but no... 
Big problem here....

**Comment by NonITExpert** [comment]
I'm seeing the same thing on Android in the github nightly version 2.3.0+fork.107.moshinda-nightly+@2024-09-02 (107)

**Comment by jotaluissoy** [comment]
And, now what!? The issue is also affecting MastodonSocial in the browser.

**Comment by rpallai** [comment]
> It's good to add that it is also happening to the Official app.

I can confirm this, tested right now. The official app doesn't show push notifications at all even on mention.

**Comment by Offerel** [comment]
Hopefully this will be solved upstream soon...

**Comment by jotaluissoy** [comment]
> Hopefully this will be solved upstream soon...

Will it be? Sure it will!?

**Comment by Offerel** [comment]
At least I hope so...

**Comment by TomCasavant** [comment]
This may be entirely coincidental but Bluesky android notifications were down as well because of something google did with their notification API which might help in diagnosing the issue

https://bsky.app/profile/did:plc:oisofpd7lj26yvgiivf3lxsi/post/3l3rckkeicb2m

**Comment by SquareDot3301** [comment]
Update : 
The official mastodon app show now push notifications but its more complexe for moshidon...

**Comment by NonITExpert** [comment]
I just started to receive notifications!

**Comment by Offerel** [comment]
Indeed. Since 2 hour's its working here to. Seems to be a serverside issue.

**Comment by NonITExpert** [comment]
Please reopen. The issue has returned

**Comment by poor-man** [comment]
Is this happening again?

**Comment by cipriani14** [comment]
I also am not receiving notifications.

**Comment by Penworks** [comment]
I'm having problems receiving notifications in one acct but not the other. On Moshidon, mastodon.social acct receives, but lor.sh does not. On my other phone I use Megalodon and push notifications work for the lor.sh acct but everything comes in marked as read so no number in the bottom icon and the mark as read top right icon is always present. The mastodon.social acct works as expected in Megalodon.

**Comment by Penworks** [comment]
(deleted previous comment)

No push notifications on Moshidon 
Push notifications work on Megalodon

Secondary issue 
Notifications mark as read icon intermittently shows up and won't go away on a second Moshidon AND Megalodon acct on different phones.
